packages/auth/src/userSession.ts

```diff
@@ -347,6 +347,16 @@ export class UserSession {
       userData.profile = tokenPayload.profile;
     }
 
+    const address = userData.profile?.stxAddress?.mainnet;
```

Comment:
how does it work with testnet?

Reply:
https://github.com/blockstack/stacks.js/blob/fe831cf28d71de9410c81cfabe256fd511e9aa4d/packages/auth/src/userSession.ts#L227-L233

The `handlePendingSignIn` function is already configured with `mainnet`. It does not have any testnet param. So the implementation is aligned with existing config. User may set `coreNode` url there is no good way to check, will it belong to `testnet` or `mainnet`. Other than that if you want to purpose anything else to support `testnet`, then most welcome.

Reply:
The hard coded URLs are only fallback. Here the user does not have a chance to use testnet.